public/app/plugins/panel/heatmap/panelcfg.gen.ts

```diff
@@ -51,15 +51,15 @@ export interface HeatmapColorOptions {
   /**
    * Sets the color mode
    */
-  mode?: HeatmapColorMode;
+  mode?: HeatmapColorMode; // TODO: remove optional when https://github.com/grafana/cuetsy/issues/74 is fixed
```

Comment:
Not sure where these comments are coming from, but it looks like the linked issue has been fixed, so the comment is not necessary anymore.

Reply:
the PR also fixed inline comment generation, so it's a comment that was already in the .cue file that's now showing up here